recipe/patches/0001-Use-pyo3-0.15.0-to-fix-cross-compilation.patch

```diff
@@ -1,25 +1,25 @@
-From c0aac7fbe675d3afcf5c64eed39f369d706f4fa1 Mon Sep 17 00:00:00 2001
-From: Chris Burr <christopher.burr@cern.ch>
-Date: Thu, 4 Nov 2021 08:33:08 +0100
+From 24584ff9e4f3b312108df4629852c3faecd1ae5f Mon Sep 17 00:00:00 2001
+From: fhoehle <fhoehle@users.noreply.github.com>
+Date: Sun, 21 Nov 2021 18:05:18 +0100
```

Comment:
Please keep the attribution in the patch as it was.

Reply:
I second this.

Reply:
Since the patch of `cryptography=35.0.0` was not applicable I generate one based on `cryptography=3.4.8` replacing the existing patch. But fine with me, to change the header, I do not think it is relevant for patching.

Reply:
Rebasing is quite a different thing from coming up with it in the first place (which often contains lots of tries that end up not getting reflected in the final commit); so keeping that information is mostly about who to ask if something doesn't work anymore (and not having to go hunt for it in the git history).